docs/index.md

```diff
@@ -70,6 +73,9 @@ In order to create the initial configuration, Relay provides the `relay config
 init` command. The command puts configuration files in the `.relay` folder
 under the current working directory:
 
+We will be creating a custom configuration since the default configuration creates a 
+`managed` mode configuration which is not supported by sentry at the current time.
```

Comment:
This paragraph can be removed, as we're creating a default configuration after all. The primary audience of this document are customers with access to managed mode.

Reply:
yep